### PR TITLE
Add symfony 5 compatibility

### PR DESCRIPTION
**Pull request by Prokyonn**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT

#### What's in this PR?

Updates dependencies for symfony 5 support.

### CODE REVIEW

File: Tests/Functional/BaseTestCase.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ArticleBundle\Tests\Functional;
+
+use Sulu\Bundle\TestBundle\Testing\SuluTestCase;
+use Symfony\Bundle\FrameworkBundle\KernelBrowser;
+
+class BaseTestCase extends SuluTestCase
```
Comment (alexander-schranz) [inline Tests/Functional/BaseTestCase.php:17]: Would not create a TestCase

File: composer.json [hunk 2]
```diff
@@ -23,34 +23,34 @@
...
-        "symfony/dotenv": "^4.3",
+        "sulu/automation-bundle": "^2.0@dev",
+        "symfony/browser-kit": "^4.3 || ^5.0",
+        "symfony/dotenv": "^4.3 || ^5.0",
         "symfony/monolog-bundle": "^3.1",
-        "symfony/stopwatch": "^4.3",
-        "zendframework/zend-stdlib": "^2.3",
-        "zendframework/zendsearch": "@dev",
+        "symfony/stopwatch": "^4.3 || ^5.0",
+        "zendframework/zend-stdlib": "^2.3 || ^3.0",
```
Comment (alexander-schranz) [inline composer.json:52]: this can be removed

**Comment by alexandersch** [comment]
Any idea when this will/can be merged?

**Comment by alexander-schranz** [comment]
@alexandersch I'm currently working on the releases of the depending dependencies. The release I plan for next week.

**Pull request merged**